### PR TITLE
pot#handleFile

### PR DESCRIPTION
**Pull request by rtomas**
Rates > POT
# function file(bytes32 what, uint256 data)
# function file(bytes32 what, address addr)

test with : yarn test

### CODE REVIEW

File: tests/mappings/modules/rates/pot/handleFile.test.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  let dataBytes = strRadToBytes(data)
+  let arg1 = tests.helpers.params.getBytes('arg2', Bytes.fromUTF8(what))
+  let arg2 = tests.helpers.params.getBytes('arg3', dataBytes)
+
+  let event = changetype<LogNote>(tests.helpers.events.getNewEvent([sig, usr, arg1, arg2]))
+
+  return event
+}
+
+function createEventVow(what: string, data: string): LogNote {
```
Comment (llopez) [inline tests/mappings/modules/rates/pot/handleFile.test.ts:25]: I looks like the only difference between the 2 event creator functions is the signature. Please pass the signature as a parameter.

File: src/mappings/modules/rates/pot.ts [hunk 2]
```diff
@@ -2,15 +2,27 @@ import { bytes, units } from '@protofire/subgraph-toolkit'
 
 import { LogNote } from '../../../../generated/Pot/Pot'
 
-import { system as systemModule } from '../../../entities'
+import { system, system as systemModule } from '../../../entities'
```
Comment (llopez) [inline src/mappings/modules/rates/pot.ts]: `system` is not been used I guess. If that is the case, please remove

**Review by llopez — CHANGES_REQUESTED** [review]

File: tests/mappings/modules/rates/pot/handleFile.test.ts (hunk 1 above)
Comment (rtomas) [reply tests/mappings/modules/rates/pot/handleFile.test.ts:25]: There are different, please check the conversion. One is from RadToBytes and the other Address.FromString

File: src/mappings/modules/rates/pot.ts (hunk 2 above)
Comment (rtomas) [reply src/mappings/modules/rates/pot.ts]: deleted it.

**Review by llopez — APPROVED** [review]

**Pull request merged**